### ISSUE TITLE
reminder (not an issue)

### ISSUE THREAD

**Issue by jerch** [shift-protocol/spec#1]
@Eugeny Hey, I have not forgotten about this, and hopefully get to review it in detail within the next 2 weeks (want to get the sixel addon back on track first).

Cheers :smiley_cat:

**Comment by jerch** [comment]
@Eugeny Is this still relevant? Saw that you integrated a different protocol? (Also note that I try to get a faster alternative to base64 up here: https://github.com/jerch/base64-sixel, which would help alot for big binary data in terminals).

**Comment by Eugeny** [comment]
Yes, it is. I haven't had time to work on it yet, but the shift branch in Tabby already supports single direction transfers.

If you mean trzsz, that's a third party integration, and that protocol offers little improvement over Zmodem IMO.

Checking out the sixel impl is still on my to-do list...
